submarine_server.py

```diff
@@ -20,13 +24,26 @@ def __init__(self, d=36, y=6, r=6, m=10, L=4, p=2):
         self.red_alert = [i % 100 for i in range(d, d+6)]
         self.submarine_time_left = self.trench_time_left = 120
         self.submarine_location = randint(0, 99)
-        self.is_submarine_in_red = self.submarine_location in self.red_alert
+        print('Waitin on port %s for players...' % PORT)
+        if gui:
+            self.is_submarine_in_red = self.submarine_location in self.red_alert
```

Comment:
uhh why is `self.is_submarine_in_red` only set if `gui` is set to `True`? That's not right...